app/assets/javascripts/ajax-video-search.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
+  $('#video-search input').keyup(function() {
+    $.get($('#video-search').attr('action'), $('#video-search').serialize(), null, 'script');
+    return false;
+  });
+});
```

Comment:
I rebased with master, but the app/ files are still there. Suggestions on how to fix this issue?